subsys/net/openthread/rpc/client/ot_rpc_thread.c

```diff
@@ -200,3 +200,61 @@ otLinkModeConfig otThreadGetLinkMode(otInstance *aInstance)
 
 	return mode;
 }
+
+uint16_t otThreadGetVersion(void)
+{
+	struct nrf_rpc_cbor_ctx ctx;
+	uint16_t version;
+
+	NRF_RPC_CBOR_ALLOC(&ot_group, ctx, 0);
+	nrf_rpc_cbor_cmd_no_err(&ot_group, OT_RPC_CMD_THREAD_GET_VERSION, &ctx,
+				nrf_rpc_rsp_decode_u16, &version);
+
+	return version;
+}
+
+const char *otThreadErrorToString(otError error)
+{
+	static const char *const error_str[OT_NUM_ERRORS] = {
```

Comment:
Why didn't just request string from the server?
I understand this seems like more optimal. However, it doesn't consider potential incompatibility if Thread code is improved.

Reply:
I personally don't like the idea of serializing something that doesn't have to be serialized. Error codes are not introduced very often so I didn't consider this to be a big problem (especially that I use `OT_NUM_ERRORS` in the array initialization so a mismatch would be detected). I can change it though.

Reply:
Ok, I have opposite opinion but do not want to block PR due to this. Let it be.